doc/WebSite/Feature-Management.md

```diff
@@ -1,21 +1,21 @@
 ### Introduction
 
 Most **SaaS** (multi-tenant) applications have **editions** (packages)
-those have different **features**. Thus, they can provide different
-**price and feature options** to thier tenants (customers).
+that have different **features**. This way, they can provide different
+**price and feature options** to their tenants (customers).
 
-ASP.NET Boilerplate provides a **feature system** to make it easier. We
+ASP.NET Boilerplate provides a **feature system** to make it easier. You
 can **define** features, **check** if a feature is **enabled** for a
-tenant and **integrate** feature system to other ASP.NET Boilerplate
+tenant, and **integrate** the feature system to other ASP.NET Boilerplate
 concepts (like [authorization](Authorization.md) and
 [navigation](Navigation.md)).
 
 #### About IFeatureValueStore
 
-Feature system uses **IFeatureValueStore** to get values of features.
-While you can implement it in your own way, it's fully implemented in
+The feature system uses the **IFeatureValueStore** to get the values of features.
+While you can implement it in your own way, it's fully implemented in the
 **module-zero** project. If it's not implemented, NullFeatureValueStore
-is used which returns null for all features (Default feature values are
+is used which returns null for all features (The default feature values are
```

Comment:
"is used ~`which returns`~ `to return` null for all features (~`The`~ `so the` default feature values are used ~`in this case`~)."